apps/web/controllers/tasks/destroy.rb

```diff
@@ -0,0 +1,22 @@
+module Web
+  module Controllers
+    module Tasks
+      class Destroy
+        include Web::Action
+
+        def call(params)
+          task = repo.find(params[:id])
+          repo.delete(task.id)
```

Comment:
we also need to remove all related TestResult records, otherwise it will fail on results page